### PR TITLE
Add documentation of per-mutator logging option

### PR DESCRIPTION
**Pull request by BackEndTea**

### CODE REVIEW

File: src/guide/usage.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ The first time you run Infection for your project, it will ask you several quest
     "logs": {
         "text": "infection-log.txt",
         "summary": "summary-log.txt",
-        "debug": "debug-log.txt"
+        "debug": "debug-log.txt",
+        "per-mutator": "per-mutator.md"
```
Comment (borNfreee) [inline src/guide/usage.md]: all our configuration keys are in camel case https://infection.github.io/guide/usage.html#Configuration-settings

what do you think about `perMutator` to make it consistent?

File: src/guide/usage.md [hunk 2]
```diff
@@ -57,6 +58,7 @@ You can commit it to the VCS and, if necessary, override it locally by creating
   * `text` - human readable text log file. Must see to understand what is going on during mutation process.
   * `summary` - summary log file, which will only tell display the amount of mutants per category, (Killed, Errored, Escaped, Timed Out & Not Covered)
   * `debug` - debug log file, which displays what mutations were found on what line, per category.
+  * `per-mutator` - a mark down file which will give a break-down of the effectiveness of each mutator.
```
Comment (borNfreee) [inline src/guide/usage.md]: Markdown is one word https://en.wikipedia.org/wiki/Markdown

**Comment by borNfreee** [comment]
Thank you @BackEndTea

**Comment by BackEndTea** [comment]
Thank you @borNfreee

**Pull request merged**